### PR TITLE
Simple view

### PR DESCRIPTION
**Pull request by yalexublue**
Basic of simple view is done, with extra components added as Navbar, Sidebar contains in the folder for future use, check out readme file inside to set up and run /stage to have a look. StageView.vue will be cleared once merge to develop is completed

### CODE REVIEW

**Comment by simaoalicechen** [comment]
I checked this during a demo session with Xu. They work great! One suggestion I have is, there was a huge overhaul of the codes sometime day, after the standup meeting. I was not informed and working according to the previous plan that we agreed upon. This led to the fact that I "wasted" my time trying to understand and coding according to what Xu showed me in the morning.  In future, I would appreciate people inform me their plan before coding, so that I can contribute accordingly. I do not have disagreements with the updated codes. They look great!

**Review by waltage — CHANGES_REQUESTED** [review]
Something is not right here -- this contains our entire history of other branches, not just the files for the simpleview

File: src/frontend/shortlist/package.json [hunk 1]
```diff
@@ -11,6 +11,8 @@
     "lint": "eslint . --ext .vue,.js,.jsx,.cjs,.mjs --fix --ignore-path .gitignore"
   },
   "dependencies": {
+    "@popperjs/core": "^2.11.6",
+    "bootstrap": "^5.2.2",
```
Comment (waltage) [inline src/frontend/shortlist/package.json]: I don't think your component needs this?  We didn't add this in yours, it was added in Pooja's

This file should be removed from this PR?

File: src/frontend/shortlist/src/components/icons/MaterialIcon.vue [hunk 2]
```diff
@@ -4,7 +4,6 @@
     props: ["src", "color", "size"],
     computed: {
       style() {
-        console.log(this.color, this.size);
         return "font-size: " + this.size + "px; " +
```
Comment (waltage) [inline src/frontend/shortlist/src/components/icons/MaterialIcon.vue]: this change should not be in this PR

File: src/frontend/shortlist/src/components/layout/NavBarTop.vue [hunk 3]
```diff
@@ -0,0 +1,33 @@
+<script>
```
Comment (waltage) [inline src/frontend/shortlist/src/components/layout/NavBarTop.vue]: this change should not be in this PR

File: src/frontend/shortlist/src/components/layout/NavLink.vue [hunk 4]
```diff
@@ -0,0 +1,33 @@
+<script>
```
Comment (waltage) [inline src/frontend/shortlist/src/components/layout/NavLink.vue]: this one needs to be removed, too

File: src/frontend/shortlist/src/components/layout/NavLinks.vue [hunk 5]
```diff
@@ -0,0 +1,46 @@
+<script>
```
Comment (waltage) [inline src/frontend/shortlist/src/components/layout/NavLinks.vue]: this one needs to be removed, too

File: src/frontend/shortlist/src/components/layout/NavLogo.vue [hunk 6]
```diff
@@ -0,0 +1,11 @@
+<script>
```
Comment (waltage) [inline src/frontend/shortlist/src/components/layout/NavLogo.vue]: this one needs to be removed, too

File: src/frontend/shortlist/src/components/layout/NavProfile.vue [hunk 7]
```diff
@@ -0,0 +1,15 @@
+<script>
```
Comment (waltage) [inline src/frontend/shortlist/src/components/layout/NavProfile.vue]: this one needs to be removed, too

File: src/frontend/shortlist/src/components/layout/sidebar/SimButton.vue [hunk 8]
```diff
@@ -0,0 +1,29 @@
+<template>
```
Comment (waltage) [inline src/frontend/shortlist/src/components/layout/sidebar/SimButton.vue:1]: I don't know where this came from, but this one doesn't have anything to do with the SimDemo stuff; did someone ask to have this included inside your PR?

File: src/frontend/shortlist/src/components/layout/sidebar/SimLeftBar.vue [hunk 9]
```diff
@@ -0,0 +1,26 @@
+<script>
```
Comment (waltage) [inline src/frontend/shortlist/src/components/layout/sidebar/SimLeftBar.vue:1]: same with this file

File: src/frontend/shortlist/src/components/layout/sidebar/SimList.vue [hunk 10]
```diff
@@ -0,0 +1,38 @@
+<template>
```
Comment (waltage) [inline src/frontend/shortlist/src/components/layout/sidebar/SimList.vue:1]: this one, too

File: src/frontend/shortlist/src/components/layout/sidebar/SimLists.vue [hunk 11]
```diff
@@ -0,0 +1,38 @@
+<template>
```
Comment (waltage) [inline src/frontend/shortlist/src/components/layout/sidebar/SimLists.vue:1]: this one

File: src/frontend/shortlist/src/components/layout/sidebar/SimRightBar.vue [hunk 12]
```diff
@@ -0,0 +1,23 @@
+<script>
```
Comment (waltage) [inline src/frontend/shortlist/src/components/layout/sidebar/SimRightBar.vue:1]: this one

File: src/frontend/shortlist/src/components/layout/sidebar/SimpleBarView.vue [hunk 13]
```diff
@@ -0,0 +1,32 @@
+<script>
```
Comment (waltage) [inline src/frontend/shortlist/src/components/layout/sidebar/SimpleBarView.vue:1]: this one... all the remaining ones in "layout/sidebar"

File: src/frontend/shortlist/src/components/preferences/PreferenceDemo.vue [hunk 14]
```diff
@@ -0,0 +1,130 @@
+<script>
```
Comment (waltage) [inline src/frontend/shortlist/src/components/preferences/PreferenceDemo.vue]: This whole preferences module is covered in Pooja's PR #88

File: src/frontend/shortlist/src/components/preferences/PreferenceDropdown.vue [hunk 15]
```diff
@@ -0,0 +1,62 @@
+<script>
```
Comment (waltage) [inline src/frontend/shortlist/src/components/preferences/PreferenceDropdown.vue]: This whole preferences module is covered in Pooja's PR #88

File: src/frontend/shortlist/src/components/layout/sidebar/SimButton.vue (hunk 8 above)
Comment (yalexublue) [reply src/frontend/shortlist/src/components/layout/sidebar/SimButton.vue:1]: This is not for SimpleView story, this is what Alice worked for side bar and trashcan

File: src/frontend/shortlist/src/components/preferences/PreferenceManager.vue [hunk 16]
```diff
@@ -0,0 +1,182 @@
+<script>
```
Comment (waltage) [inline src/frontend/shortlist/src/components/preferences/PreferenceManager.vue]: everything from preferences should be removed from this PR

**Review by waltage — APPROVED** [review]
LGTM

**Pull request merged**